src/node_options.cc

```diff
@@ -50,11 +50,13 @@ void DebugOptions::CheckOptions(std::vector<std::string>* errors,
                       "`node --inspect-brk` instead.");
   }
 
-  std::vector<std::string> destinations =
-      SplitString(inspect_publish_uid_string, ',');
+  std::vector<std::string_view> destinations =
+      SplitString(std::string_view(inspect_publish_uid_string.data(),
+                                   inspect_publish_uid_string.size()),
+                  ",");
```

Comment:
The `","` has to be cast to a std::string_view.

If you want to be explicit, you can use the `","sv` syntax

https://en.cppreference.com/w/cpp/string/basic_string_view/operator%22%22sv